miniupnpd/Makefile.linux

```diff
@@ -95,13 +95,15 @@ $(info please install uuid-dev package / libuuid)
 endif # ($(TEST),1)
 endif # ($(TARGET_OPENWRT,)
 
+ifneq ($(shell --ldd --version | grep GLIBC),)
```

Comment:
it should be `$(shell ldd --version | grep GLIBC)`

Reply:
as it is :
```
/bin/sh: 0: Illegal option --
```

Reply:
doh... ofc..